shell/platform/windows/window_binding_handler_delegate.h

```diff
@@ -135,6 +136,9 @@ class WindowBindingHandlerDelegate {
 
   // Update the status of the high contrast feature
   virtual void UpdateHighContrastEnabled(bool enabled) = 0;
+
+  // Obtain a pointer to the fragment root delegate.
```

Comment:
It may be useful to add a little detail on how this is used by UIA, since it's not something that was required in MSAA.